logprep/util/rule_dry_runner.py

```diff
@@ -54,8 +54,8 @@ def get_patched_runner(config_path, logger):
 class DryRunner:
```

Comment:
this is the dark side of logprep. We need to refactor this class. Could you spend some time to increase test coverage for the `run` method?

Reply:
I started to do some refactorings here. It is still not perfect but I think its an improvement already. Similar to my other remark I would prefer though not to spend much more time on this as it started as a bugfix and wasn't intended as a huge refactoring.

Reply:
good job, but 100% test coverage  before doing refactorings should be the better way.
I keep my fingers crossed 🤞